src/kontrol/prove.py

```diff
@@ -978,101 +987,104 @@ def _create_cse_accounts(
     new_accounts: list[KInner] = []
     new_account_constraints: list[KApply] = []
 
-    storage_map = KVariable(contract_name + '_STORAGE', sort=KSort('Map'))
-    new_accounts.append(Foundry.symbolic_account(contract_name, contract_code, storage_map))
+    storage_map: KInner = KVariable(contract_name + '_STORAGE', sort=KSort('Map'))
+
+    singly_occupied_slots = [
+        slot for (slot, count) in Counter([field.slot for field in storage_fields]).items() if count == 1
+    ]
```

Comment:
Currently, we are limited to handling only the cases in which addresses and contracts in storage occupy an entire slot. This should be one of the immedate next PRs.